resources/backup_scripts/db-backup.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python
+
+import os
+import glob
+import json
+import re
+import argparse
+from datetime import date, timedelta
+from collections import namedtuple
+
+def get_settings():
+    # Default settings
+    s3_bucket = 'warmshowers-database-backup'
+    backup_dir = '/var/backups/db_backups'
+    dry_run  = False
+
+    # Parse command-line arguments
+    parser = argparse.ArgumentParser(description='Sync database dumps from the last seven days.')
```

Comment:
> We might want more than seven days. Is this a requirement from @chrisarusso?

Yes, @chrisarusso would like a seven day backup, with an emphasis that restoring from the previous day is top priority.

> Do we want to enable versioning on the bucket to be able to retrieve any DB that's been written to S3?

That seems like a good idea. We should set a "noncurrent expiration lifecycle policy" so that the versioned copies eventually get deleted after our script deletes them.